modules/mri_violations/test/mri_violationsTest.php

```diff
@@ -167,6 +168,7 @@ public function setUp()
              'series_description' => 'Test Series Description',
              'SeriesUID'          => '5556',
              'TarchiveID'         => '264',
+             'MriProtocolGroupID' => 1,
```

Comment:
before this insert, 
        $this->DB->insert(
            "mri_protocol_group",
            array(
             'MriProtocolGroupID' => '11',
             'Name'               => 'test',
            )
        );

Reply:
in the tearDown function, 
        $this->DB->delete(
            "mri_protocol_group",
            array('Name' => 'test')
        );